server/routes/signup.js

```diff
@@ -0,0 +1,25 @@
+let express = require('express');
+let router = express.Router();
+
+const User = require('../models/user')
+
+router.route('/')
+  .get((req, res, next) => {
+    res.setHeader('Content-Type', 'application/json');
+    res.send('Test response');
+  })
+
+  .post((req, res) => {
```

Comment:
we should do more validations here before adding a user. Also remember to hash the password.
We also need some kind of token system for keeping a user logged in. Read the details of ticket #17 carefully.
If these are not supposed to be in this PR then try to keep the scope limited and keep these things for another PR

Reply:
Oh, I only meant for this PR to be for the front-end and mongodb connection. I misunderstood what I was to push. I have not completed #17 yet.